### PR TITLE
feat: Added Light mode to Profile Page

### LINKED ISSUE #434: Profile page should have light/dark mode
## Context

So we can give a better user experience, we are working on a light/dark mode.

## Expected Behavior

We use Tailwind so we should enable this by having a default light mode and dark mode with the `dark:` prefix. 
You can see how this is done here: https://tailwindcss.com/docs/dark-mode

## Current Behavior

By default, we only use a dark theme. 
You can see the feed here: https://www.codu.co/settings

## Additional info

The background for light mode should be: `bg-neutral-100`
The text should be: `text-neutral-900`
The cards should have a background of: `bg-white`
borders for light mode should be: `neutral-100`

If there is more questions let me know as I'm sure I've missed something!

### PR DESCRIPTION
**Pull request by pateldivyesh1323**
# ✨ Codu Pull Request 💻

Codu Logo

## Pull Request details:
- Fixes : #434 
- Add Light mode to Profile page
## Any Breaking changes:
- None

## Associated Screenshots:

image
image

### CODE REVIEW

File: components/ArticlePreview/ArticlePreview.tsx [hunk 1]
```diff
@@ -160,20 +160,20 @@ const ArticlePreview: NextPage<Props> = ({
                   leaveFrom="transform opacity-100 scale-100"
                   leaveTo="transform opacity-0 scale-95"
                 >
-                  <Menu.Items className="origin-top-right absolute bottom-10 right-0 mt-2 w-48 rounded-md shadow-lg py-1 bg-white ring-1 px-1 ring-black ring-opacity-5 focus:outline-none">
+                  <Menu.Items className="origin-top-right absolute bottom-10 right-0 mt-2 w-48 rounded-md shadow-lg py-1 bg-black ring-white dark:bg-white ring-1 px-1 dark:ring-black ring-opacity-5 focus:outline-none">
```
Comment (NiallJoeMaher) [inline components/ArticlePreview/ArticlePreview.tsx]: Could you reverse the styles and screenshot what it looks like? 
I didn't even think about the dark mode for the menu options. 🧠 

It looks good but I think it makes more sense to have a dark manu on the dark mode (and not the light mode). Thanks! 🙌

**Review by NiallJoeMaher — CHANGES_REQUESTED** [review]

File: components/ArticlePreview/ArticlePreview.tsx (hunk 1 above)
Comment (pateldivyesh1323) [reply components/ArticlePreview/ArticlePreview.tsx]: @NiallJoeMaher It looks like this.

image

Comment (NiallJoeMaher) [reply components/ArticlePreview/ArticlePreview.tsx]: This looks great! Let's get that pushed and I'll approve.

**Review by NiallJoeMaher — APPROVED** [review]
LGTM! ✨

**Pull request merged**